modules/ossm-migrating-from-ior-to-explicitly-managed-routes.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-route-migration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-from-ior-to-explicitly-managed-routes_{context}"]
+= Migrating from Istio OpenShift Routing to explicitly-managed routes
+
+This procedure explains how to disable Istio OpenShift Routing (IOR) in {SMProductName}, and how to continue to use and manage Routes that were originally created using IOR. This procedure also provides an example of how to explicitly create a new Route targetting an existing gateway Service.
```

Comment:
Not sure if there's something special at play for OSSM, but I notice a lot of init-capped "Service" throughout the PR, which seem to be in reference to `Service` objects. Typically these would be either "service" (lowercase, no markup) or "`Service` object" per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), e.g.:
```suggestion
This procedure explains how to disable Istio OpenShift Routing (IOR) in {SMProductName}, and how to continue to use and manage Routes that were originally created using IOR. This procedure also provides an example of how to explicitly create a new Route targetting an existing gateway `Service` object.
```
If this is adjusted (either now or in a follow-up PR), apply uniformly throughout the assemblies/modules/PR.

Reply:
attempted to fix in these documents. Created OSSM-6911 to investigate further in the broader doc set.